commercetools/commercetools-sdk-java-api/src/integrationTest/java/commercetools/category/CategoryQueryTests.java

```diff
@@ -0,0 +1,54 @@
+
+package commercetools.category;
+
+import static commercetools.category.CategoryFixtures.*;
+
+import com.commercetools.api.models.category.Category;
+import com.commercetools.api.models.category.CategoryPagedQueryResponse;
+import commercetools.utils.CommercetoolsTestUtils;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class CategoryQueryTests {
+    @Test
+    public void getById() {
+        withCategory(category -> {
+            Category queriedCategory = CommercetoolsTestUtils.getProjectApiRoot()
+                    .categories()
+                    .withId(category.getId())
+                    .get()
+                    .executeBlocking()
+                    .getBody();
+            Assertions.assertEquals(category.getId(), queriedCategory.getId());
```

Comment:
Just seeing you are using here junit Assertions. For other tests we used AssertJ.

Maybe we should align to one assertion framework